2-With-broker/UserDetailsClient/UserDetailsClient.UWP/UserDetailsClient.UWP.csproj

```diff
@@ -128,7 +128,7 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Identity.Client" Version="4.45.0" />
+    <PackageReference Include="Microsoft.Identity.Client" Version="[4.46.0-preview2]" />

```

Comment:
did you put in the version manually or did nuget do it? Interesting use of floating versions.

Reply:
It was done by NuGet.